Source/WebKit/UIProcess/mac/WKImmediateActionController.mm

```diff
@@ -393,16 +394,17 @@ - (NSRect)menuItem:(NSMenuItem *)menuItem itemFrameForPoint:(NSPoint)point
     if (!hitTestResult)
         return NSZeroRect;
 
-    return [_view convertRect:hitTestResult->elementBoundingBox() toView:nil];
+    return [_view.get() convertRect:hitTestResult->elementBoundingBox() toView:nil];
```

Comment:
@rniwa I'm sorry, it looks like I had forgotten to fix this one, r? again please :)